src/cookbook/plugins/achievements-leaderboard.md

```diff
@@ -0,0 +1,350 @@
+        official [Play Games Services guide](https://developers.google.com/games/services/console/enabling).    
+         
+        ![Screenshot showing the Games Services section in Google Play Console](/assets/images/docs/cookbook/play-console-play-games-services.png)
+
+3.  When done, you can start adding leaderboards and achievements in
+    **Play Games Services** → **Setup and management**. Create the exact
+    same set as you did on the iOS side. Make note of IDs.
+
+4.  Go to **Play Games Services → Setup and management → Publishing**.
+
+5.  Click **Publish**. Don't worry, this doesn't actually publish your
+    game. It only publishes the achievements and leaderboard. Once a
+    leaderboard, for example, is published this way, it cannot be
+    unpublished.
+
+6.  Go to **Play Games Services** **→ Setup and management →
+    Configuration → Credentials**.
+
+7.  Find the **Get resources** button. It returns an XML file with the
+    Play Games Services IDs.
+
+    ```xml
+    <!-- THIS IS JUST AN EXAMPLE -->
+    <?xml version="1.0" encoding="utf-8"?>
+    <resources>
+        <!--app_id-->
+        <string name="app_id" translatable="false">424242424242</string>
+        <!--package_name-->
+        <string name="package_name" translatable="false">dev.flutter.tictactoe</string>
+        <!--achievement First win-->
+        <string name="achievement_first_win" translatable="false">sOmEiDsTrInG</string>
+        <!--leaderboard Highest Score-->
+        <string name="leaderboard_highest_score" translatable="false">sOmEiDsTrInG</string>
+    </resources>
+    ```
+
+8.  Add a file at `android/app/src/main/res/values/games-ids.xml`
+    containing the XML you received in the previous step.
+
+## 2. Sign in to the game service
+
+Now that you have set up *Game Center* and *Play Games Services*, and
+have your achievement & leaderboard IDs ready, it's finally Dart time.
+
+1.  Add a dependency on the [`games_services` package](https://pub.dev/packages/games_services).
+
+    ```shell
+    flutter pub add games_services
+    ```
+
+2.  Before you can do anything else, you have to sign the player into
+    the game service.
+
+    ```dart
+    try {
+      await GamesServices.signIn();
+    } on PlatformException catch (e) {
+      // ... deal with failures ...
+    }
+    ```
+
+The sign in happens in the background. It takes several seconds, so
+don't call `signIn()` before `runApp()` or the players will be forced to
+stare at a blank screen every time they start your game.
+
+The API calls to the `games_services` API can fail for a multitude of
+reasons. Therefore, every call should be wrapped in a try-catch block as
+in the previous example. The rest of this recipe omits exception
+handling for clarity.
+
+**Note:** It's a good practice to create a controller. This would be a
+`ChangeNotifier`, a bloc, or some other piece of logic that wraps around
+the raw functionality of the `games_services` plugin.
```

Comment:
```suggestion
{{site.alert.note}}
  It's a good practice to create a controller. This would be a
  `ChangeNotifier`, a bloc, or some other piece of logic that wraps around
  the raw functionality of the `games_services` plugin.
{{site.alert.end}}
```